.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
+}
```

Comment:
@Awinja-Andela , I think we should have an extra line at the end of the file. Even git recommends